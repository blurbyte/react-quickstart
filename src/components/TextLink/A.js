import styled from 'styled-components';

const A = styled.a`
  color: #ff4326;

  &:hover {
    text-decoration: underline;
  }
`;

export default A;
