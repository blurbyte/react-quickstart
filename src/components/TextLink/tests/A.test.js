import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import A from '../A';

describe('<A />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <A />
    );

    expect(component.toJSON()).toMatchStyledComponentsSnapshot();
  });
});
