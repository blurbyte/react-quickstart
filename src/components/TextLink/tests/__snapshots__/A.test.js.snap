// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<A /> should render correctly 1`] = `
.ifuQuG {
  color: #ff4326;
}

.ifuQuG:hover {
  text-decoration: underline;
}

<a
  className="ifuQuG"
/>
`;
